Fabric.Authorization.AccessControl/src/app/modules/access-control/custom-group/custom-group.component.html

```diff
@@ -3,9 +3,18 @@ <h3 class="h3">
         <span>Custom Group Name</span>
     </h3>
     <div class="input-combo">
-        <input hc-input class="hc-input" required [(ngModel)]="groupName" [readOnly]="editMode" [highlight]="groupNameinvalid" (keyup)="groupNameSubject.next($event.target.value)">
+        <input hc-input class="hc-input" required [(ngModel)]="groupName" [readOnly]="editMode" [highlight]="groupNameinvalid" (keyup)="groupNameSubject.next($event.target.value)" placeholder="enter group name">
+        <i [class]="searchingGroup ? 'btn-icon fa fa-spinner fa-pulse fa-fw' : 'btn-icon fa fa-search'"></i>
         <div class="form-errors" *ngIf="groupNameinvalid" [innerText]="groupNameError"></div>
     </div>
+    <div class="scroll-list">
+        <div *ngIf="customGroups.length > 0"><span>Select an existing group:</span></div>
+        <ul>
+            <li *ngFor="let group of customGroups">
```

Comment:
not using the scroll-table will make it not consistent with the member page. I am ok converting over that component to using <li> but I would prefer them to be consistent.

Reply:
Ok sure, I can change the scroll list to be a scroll-table